src/st/dai/e_bone_pillar.c

```diff
@@ -0,0 +1,489 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "dai.h"
+
+#ifdef VERSION_PSP
+s32 D_us_80181C54[] = {0x00200000, 0x00040000, 0xFFFC0004, 0x0000FFF8};
+#else
+s32 D_us_80181C54[] = {
+    0x00200000, 0x00040000, 0xFFFC0004, 0x0000FFF8, 0x00080000, 0x00040000,
+    0xFFFC0002, 0x0000FFFC, 0x00240000, 0x00040000, 0xFFFC0002, 0x0000FFFC};
+#endif
+s32 D_us_80181C84[] = {0x000E0000, 0x00000000};
+s16 D_us_80181C8C[] = {
+    0x0002, 0xFFF1, 0x0002, 0xFFEF, 0xFFFC, 0x000A, 0xFFFC, 0x0009};
+s16 D_us_80181C9C[] = {0x0026, 0x0026, 0x000E, 0x000A};
+s16 D_us_80181CA4[] = {0x02C0, 0x0540, 0x0AC0, 0x0D40};
+s32 D_us_80181CAC[] = {
+    0x02020130, 0x02020102, 0x02020102, 0x02020102, 0x02020102,
+    0x04020302, 0x04050502, 0x04020602, 0x07020602, 0x07020602,
+    0x07020802, 0x07020802, 0x04110602, 0x000000FF};
+s32 D_us_80181CE4[] = {
```

Comment:
Similarly to sozud's comment above, these are `AnimateEntity` arguments so should be `u8` arrays